doc-content/enterprise-only/release-notes/rn-known-issues-ref-stash.adoc

```diff
@@ -0,0 +1,263 @@
+[id='rn-known-issues-con']
+= Known issues
+
+This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
+
+
+ifdef::DM[]
+
+endif::[]
+
+== {CENTRAL}
+
+.You cannot add a new group in {CENTRAL} integrated with RH-SSO [https://issues.jboss.org/browse/RHPAM-2036[RHPAM-2036]]
+
+Issue: The following exception occurs when you try to add a new group in {CENTRAL} if {CENTRAL} is integrated with RH-SSO 7.3.0:
+[source]
+----
+java.lang.NoSuchMethodError:org.keycloak.representations.idm.RoleRepresentation.setScopeParamRequired(Ljava/lang/Boolean;)
+----
+
+Steps to reproduce:
+
+. Open {CENTRAL}.
+. Open the *Admin* page.
+. Open *Groups*.
+. Try to create a new group.
+
+
+Expected result: The new group is created.
+
+Actual result: A dialog box with the listed exception appears.
+
+Workaround: Add the group in RH-SSO instead of adding it in {CENTRAL}.
+
+.Human task comments are not retained when a process instance is finished
+//[https://issues.jboss.org/browse/JBPM-8556[JBPM-8556]]
+
+Issue: Human task comments are retained and visible only while the respective process instance is running. When the process instance is completed or aborted, comments from human tasks disappear even though the tasks are still visible in {CENTRAL}.
+
+Steps to reproduce:
+
+. Start a process instance with two consecutive human tasks.
+. Comment on the first human task.
+. Claim, start, and complete the first task.
+. Check the log file to confirm that the comments are retained for the completed task.
```

Comment:
It is not log file (here and in 'Excpected' and 'Actual' results below), but rather a Tab in UI, can we refer to it as Comments Tab (of a Human Task)?

Reply:
Done